test/remote/gateways/remote_creditcall_test.rb

```diff
@@ -0,0 +1,147 @@
+require 'test_helper'
+
+class RemoteCreditcallTest < Test::Unit::TestCase
+  def setup
+    @gateway = CreditcallGateway.new(fixtures(:creditcall))
+
+    @amount = 100
+    @credit_card = credit_card('4000100011112224')
+    @declined_card = credit_card('4000300011112220')
+    @options = {
+      billing_address: address,
+      description: 'Store Purchase'
+    }
+  end
+
+  def test_successful_purchase
+    response = @gateway.purchase(@amount, @credit_card, @options)
+    assert_success response
+    assert_equal 'Succeeded', response.message
+  end
+
+  def test_successful_purchase_with_more_options
+    options = {
+      order_id: '1',
+      ip: "127.0.0.1",
+      email: "joe@example.com"
+    }
+
+    response = @gateway.purchase(@amount, @credit_card, options)
+    assert_success response
+    assert_equal 'Succeeded', response.message
+  end
+
```

Comment:
Could be :cool: to have a test to run a purchase without a `billing_address` and `description`.  Perhaps a `test_successful_purchase_sans_options`.